README.md

```diff
@@ -28,11 +28,66 @@ Then you should see something like this in the terminal, showing you can see the
 npm test
 ```
 
+```bash
```

Comment:
Just noticed something: the ```bash.....``` in the beginning of the  and end whole block shouldnt be there.
Its causing the whole ext to be misformated